### PR TITLE
improvement: use package:web

### PR DESCRIPTION
**Pull request by schultek**
## Type of Change

- ❌ Breaking change
- ✨ New feature or improvement

### CODE REVIEW

**Comment by simolus3** [comment]
@schultek Would a contribution to finish this work be welcome or are you actively working on this?

**Comment by schultek** [comment]
Hi, contributions are always appreciated!

For this one the implementation is fairly finished, I just need to fix the tests and also do some more testing in general.

If you want to help, a review or testing would be amazing.

File: packages/jaspr/lib/src/foundation/constants.dart [hunk 1]
```diff
@@ -71,7 +71,7 @@ const double precisionErrorTolerance = 1e-10;
 /// backed by the same kind of object. Thus a double `0.0` is identical
 /// to an integer `0`. This is not true for Dart code running in AOT or on the
 /// VM.
-const bool kIsWeb = identical(0, 0.0);
+const bool kIsWeb = bool.fromEnvironment('dart.library.js_util');
```
Comment (simolus3) [inline packages/jaspr/lib/src/foundation/constants.dart]: Legacy interop libraries are set to be removed, so I think it's better to use `dart.library.js_interop` for this to work in the future.

Comment (schultek) [reply packages/jaspr/lib/src/foundation/constants.dart]: True. I just copied this over from flutter without thinking.

Comment (schultek) [reply packages/jaspr/lib/src/foundation/constants.dart]: Done

**Pull request merged**